src/zenml/zen_stores/migrations/versions/5994f9ad0489_introduce_role_permissions.py

```diff
@@ -0,0 +1,121 @@
+"""Introduce role permissions [5994f9ad0489].
+
+Revision ID: 5994f9ad0489
+Revises: c1b18cec3a48
+Create Date: 2022-10-25 23:52:25.935344
+
+"""
+import datetime
+import uuid
+
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5994f9ad0489"
+down_revision = "ccd68b7825ae"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    op.create_table(
+        "rolepermissionschema",
+        sa.Column("name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("role_id", sqlmodel.sql.sqltypes.GUID(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["role_id"],
+            ["roleschema.id"],
+        ),
+        sa.PrimaryKeyConstraint("name", "role_id"),
+    )
+    # get metadata from current connection
+    meta = sa.MetaData(bind=op.get_bind())
```

Comment:
I didn't realize how painful it can be to populate the database without SQLModel. I thought you might use SQLModel here to add the default roles and update all existing users instead of alembic and raw sqlalchemy, but then I remembered that SQLModel schemas are a living thing that may change even more in the future.